main.js

```diff
@@ -111,13 +120,15 @@ function showText() {
 inputField.addEventListener('keydown', e => {
   // Add wrong class to input field
   switch (typingMode) {
+		case 'custom':
     case 'wordcount':
       if (currentWord < wordList.length) inputFieldClass();
     case 'time':
       if (timerActive) inputFieldClass();
   }
   function inputFieldClass() {
-    if (e.key >= 'a' && e.key <= 'z' || (e.key === `'` || e.key === ',' || e.key === '.' || e.key === ';')) {
+    const puncRegex = /[!:;'",.\/?!@#$%^&*()_}{\[\]-]/;

```

Comment:
```suggestion
    const puncRegex = /[!:;'",.\/?!@#$%^&*()_}{\[\]-\+\=]/;
```

Add `+` and `=` to match support for said characters in [`betterSplit()`](https://github.com/briano1905/typings/pull/68/files#r421577328).

Should the regex for punctuation marks be kept as a global constant?

Reply:
That’s a good idea, since it should be the same for the whole app.